docs/MetadataProps.md

```diff
@@ -0,0 +1,34 @@
+# Metadata
+# Metadata
+
+In addition to the core metadata recommendations listed in the [extensibility documentation](IR.md#metadata) there is additional experimental metadata to help provide information for model inputs and outputs.  
```

Comment:
This doc is called metadata propagation, so we should not introduce the additional experimental metadata here.
Let's move the introduction of the experimental metadata to metadata section in IR doc.

Reply:
The example should help to show that we have 3 separate pieces here, and how to tie them all together.